### DISCUSSION TITLE
can I install .deb file from ubuntu?

### DISCUSSION THREAD

**Discussion by aicynide** [termux/termux-packages#16679] [Q&A] [answered]
Or does it need to modifed for android?

**Comment by sylirre** [comment, marked as answer]
Termux is a distribution not compatible with Ubuntu, Debian or other apt/dpkg based distributions: https://wiki.termux.com/wiki/Differences_from_Linux. It uses own .deb files hosted at https://packages.termux.dev/ and its mirrors.

Use of `.deb` format means nothing in the context of distribution compatibility. This is only archive format, like zip or tar. Usage of .deb files doesn't mean that distribution is Debian-based.
